common/lib/api-client/models.js

```diff
@@ -176,4 +180,25 @@ module.exports = {
       filesize: '',
     },
   },
+  allocation: {
+    attributes: {
+      from_location: {
+        jsonApi: 'hasOne',
+        type: 'locations',
+      },
+      to_location: {
+        jsonApi: 'hasOne',
+        type: 'locations',
+      },
+      date: '',
+      prisoner_category: '',
+      sentence_length: '',
+      complex_cases: [],
+      moves_count: 0,
+      complete_in_full: '', // boolean
```

Comment:
This comment feels like a bit of a red flag that maybe we should update the name of the variable? I find a `is_` prefix helps to suggest when something is a Boolean and makes the code much more readable. 

So this could become `is_complete_in_full`. 

Without the “is” this variable could be read as “to” complete in full when used within the app.

Reply:
Could this be changed to `complete_in_full: false` instead? I'd prefer not to rename the back end data model attributes at this stage - also I'm not 100% sure that "is complete in full" makes sense - my understanding is that this attribute indicates that the allocation must be completed in full (i.e. if there are 3 prisoners specified in `moves_count` then 3 prisoners must be added to the allocation). It's not meant to be an indication of the move or allocation actually having been completed.